### PR TITLE
remove messages at startup, fix #258

### LINKED ISSUE #258: Remove info texts on startup
Hi,

Everytime I start habdroid I get two info messages:
- connecting to configured url
- voice recognizer not present...

In my opinion it would be better, when the first messages shows up once after changing url, the second as a popup with "never show this again" and "ok" button

### PR DESCRIPTION
**Pull request by mueller-ma**
IMO these messages are not needed:
1. When I am on WIFI and see my sitemap, it connected most likely direct and when I am not on WIFI and I see my sitemap, it connected via Cloud Companion. It still logs the messages and gives and error when connection failed.
2. If the voice recognizer is not available the microphone icon is not present

### CODE REVIEW

**Comment by digitaldan** [comment]
Hi can you resolve the conflicts?  Thanks!

**Comment by mueller-ma** [comment]
@digitaldan Done

**Comment by mueller-ma** [comment]
Dont merge it. The app crashes now when starting the second time

**Comment by mueller-ma** [comment]
@digitaldan I am confused with Android Studio. I built the same version again and it runs without any problems

**Comment by digitaldan** [comment]
I'll test this out as soon as a figure out our build server issues.

**Comment by digitaldan** [comment]
Build server is finally fixed, looking at this now.

File: mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public void start() {
                     // If remote URL is configured
                     if (mOpenHABUrl.length() > 0) {
                         Log.d(TAG, "Connecting to remote URL " + mOpenHABUrl);
-                        openHABTracked(mOpenHABUrl, mCtx.getString(R.string.info_conn_rem_url));
```
Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: What is the purpose for changing this?

Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: It removes the "Connected to remote URL" toast when starting the app

Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: ok, so why do we not want to give feedback to the user that the app is trying to connect to a URL?   I have seen where the app tries the local URL (which is not available) then tries the remote URL,  I think it's good that we give some feedback especially b/c in this case it adds a delay to loading a sitemap. Or is this something else?

Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: I find the notifications anoying. When I'm in my wifi it connects to local url, if not to the remote url. I don't need feedback for that.
Every time I open the app I get these notifications and I get the "voice recognizer..." every time I rotate the phone

Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: I agree with the voice recognizer , my guess is that it is a very rarely used feature and probably extra noise, but I don't have an issue with the connecting messages.  I don't think I have heard others complain about it so before we make this kind of change I would prefer we have a chorus of feedback saying this is undesirable.  I actually prefer to know if I'm connecting locally or remotely.

Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: The voice recognizer problem appears only when the google app is not available

Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: Can we modify the PR to not complain about the voice recognizer but leave the other messages?

Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: So split it in two PRs?

Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: 2 pr's would be good.  I'm not sure we want to change the local/remote notifications however.

Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java]: See #303

**Pull request closed without merging**